app/javascript/packs/components/LaneHeader.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default class LaneHeader extends React.Component {
+  render() {
+    return (
+      <div>
+        <b>{this.props.id}</b> ({this.props.cards.length}/{this.props.total_count})
```

Comment:
Use destructuring for `props` & `state` like
`const { id, cards, totalCount } = this.props;`

No snake_case